app/code/Magento/InventoryCatalogAdminUi/Model/BulkOperationsConfig.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalogAdminUi\Model;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+class BulkOperationsConfig
```

Comment:
Our current implementation represented by just Controllers, but potentially it's possible that some will wish to use SourceItem transfer via WebAPI
so, that we need to move BulkOperationsConfig and XML configuration from `AdminUi` to InventoryCatalog.

As there is a possibility that  MSI would be used in a headless way (all AdminUI modules in this case would be dismantled)

Reply:
That configuration only affects the backend experience, they are not related to the API calls.
My idea was that one integrator using the API calls could use the standard bulk/api.
If we want to move at interface level I would introduce a new optional parameter in the call with the bulk size.

Reply:
can we consider a situation when somebody will want to send  SourceItems to transfer via Web API , and not to wait synchronously while Magento will process this migration. So that 202 HTTP header (Accpted) would be returned? 

I am just considering that operation type  **Synchronous / Async** is not a responsibility of **AdminUI** module